app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rate/services.phtml

```diff
@@ -7,10 +7,11 @@
 <?php
```

Comment:
please remove not needed closing and opening PHP tags

Reply:
fixed